.rubocop.yml

```diff
@@ -6,3 +6,6 @@ inherit_gem:
 inherit_mode:
   merge:
     - Exclude
+
+Rails/SaveBang:
```

Comment:
My preference would be to not disable anything here.

Reply:
see below, we need all valid postcodes in the seed data

Reply:
I understand, we can definitely have only valid postcodes in the seed data. But why can't we save with the `create!` that's strange. In any case the seeder of data is not really a "production" code, so it's probably fine, but I'd just disable that one particular Rubocop breach, rather then everything. 

You can do ` # rubocop:disable Rails/SaveBang` on that line 🙏

Reply:
coolio, done